### PR TITLE
[ntuple, tutorials] Drop tree conversion from ntpl004

### PR DESCRIPTION
**Pull request by jblomer**
Makes the ntpl004_dimuon tutorial more focused on the RDataFrame aspect. No distraction due to the tree conversion step, the RNTuple file is pre-generated and available on root.cern.ch/files.

### CODE REVIEW

File: tutorials/v7/ntuple/ntpl004_dimuon.C [hunk 1]
```diff
@@ -1,9 +1,10 @@
 /// \file
 /// \ingroup tutorial_ntuple
 /// \notebook
-/// Convert CMS open data from a TTree to RNTuple.
-/// This tutorial illustrates data conversion and data processing with RNTuple and RDataFrame.  In contrast to the
-/// LHCb open data tutorial, the data model in this tutorial is not tabular but entries have variable lengths vectors
+/// Mini-Analysis on CMS OpenData with RDataFrame.
+/// This tutorial illustrates that there is analyzing data with RDataFrame works the same
```
Comment (vepadulano) [inline tutorials/v7/ntuple/ntpl004_dimuon.C:5]:
```suggestion
/// This tutorial illustrates that analyzing data with RDataFrame works the same
```

Comment (vepadulano) [reply tutorials/v7/ntuple/ntpl004_dimuon.C:5]: Possibly small typo

**Comment by pcanal** [comment]
Do we have a test for the conversion?   Should we have a standalone tutorials for the conversion or do we expect that most user will create RNTuple from scratch?

**Comment by jblomer** [comment]
@pcanal @Axel-Naumann The idea was to explicitly take out the relatively complicated conversion step from the tutorials in order to not drive away users.  So I think we don't want to add this as another tutorial.  Regarding testing, I also don't think this code would test anything specific that is not covered by other tests.

**Comment by pcanal** [comment]
> The idea was to explicitly take out the relatively complicated conversion step from the tutorials in order to not drive away users. So I think we don't want to add this as another tutorial.

Yes, I agree that having a simple tutorials for reading is better.  If we expect user to have-to/want to convert TTree into RTNtuple then we should consider have a tutorials, shouldn't we?

**Comment by oshadura** [comment]
My two cents, I agree that in case of tutorial, removing conversion makes it much more attractive and understandable, but I would still preserve a conversion as a separate mini tutorial...

**Pull request merged**